lib/Language/Haskell/Stylish/Config.hs

```diff
@@ -260,5 +259,7 @@ parseTrailingWhitespace _ _ = return TrailingWhitespace.step
 
 --------------------------------------------------------------------------------
 parseUnicodeSyntax :: Config -> A.Object -> A.Parser Step
-parseUnicodeSyntax _ o = UnicodeSyntax.step
-    <$> o A..:? "add_language_pragma" A..!= True
+parseUnicodeSyntax c o = UnicodeSyntax.step
+    <$> o A..:? "add_language_pragma"    A..!= True
+    <*> o A..:? "allow_lowercase_pragma" A..!= allowLowercase
+    where allowLowercase = elem "allow_lowercase" $ configLanguageExtensions c
```

Comment:
I have serious doubts about this part, @jaspervdj is there a better way to access config flags from other parts?